### ISSUE TITLE
LOSS stuck arount 0.69

### ISSUE THREAD

**Issue by ashishjamarkattel** [primepake/better_wav2lip#114]
I got an strange issue while training color_sync net.
I have read your other issue for filtering the dataset. to [-1, 1 ] i really dont understand this. How do we do that.

1. I have cut the video into less than 5sec.
2. resolution of video is 760px to 1080px.
3. Lr is 1e-4 also tried 1e-5
4. batch size is 16.

**Comment by NochnoyRitzar** [comment]
Hi! 
`[-1, 1]` refers to are offset values between audio and video that you get after using code from [https://github.com/joonson/syncnet_python](url) on your videos.

I recommend running `run_pipeline.py` first, as it preprocesses video to make it suitable for syncnet, and then running `run_syncnet.py` from the same repo

**Comment by ashishjamarkattel** [comment]
Thanks for the help. Is running those 2 command is enough ?

**Comment by NochnoyRitzar** [comment]
Yes. There is another one which is `run_visualize.py`, but it's just for visualization purpose. 
BTW I see that the link is bad in previous comment, here is a correct one - https://github.com/joonson/syncnet_python

**Comment by 397688551** [comment]
I have encountered the same problem, which is also not falling at 0.69. May I ask if you have solved it？

**Comment by ashishjamarkattel** [comment]
> I have encountered the same problem, which is also not falling at 0.69. May I ask if you have solved it？

I am working on it. But i am not able to solve it still. Somewhere in case of mine syncnet_python not working for my avspeech dataset. Please let me know if you have solve it.

**Comment by drakitLiu** [comment]
> > I have encountered the same problem, which is also not falling at 0.69. May I ask if you have solved it？
> 
> I am working on it. But i am not able to solve it still. Somewhere in case of mine syncnet_python not working for my avspeech dataset. Please let me know if you have solve it.
I have encountered the same problem, which is also not falling at 0.69. oh my ! May I ask if you have solved it？

**Comment by drakitLiu** [comment]
> May I ask if you have solved it？

 May I ask if you have solved it？

**Comment by drakitLiu** [comment]
> > I have encountered the same problem, which is also not falling at 0.69. May I ask if you have solved it？
> 
> I am working on it. But i am not able to solve it still. Somewhere in case of mine syncnet_python not working for my avspeech dataset. Please let me know if you have solve it.

Hi ! did  you have solved it？
